### DISCUSSION TITLE
问个关于版本的问题

### DISCUSSION THREAD

**Discussion by FangFuJi1991** [Meloong-Git/PCL#1644] [提问] [answered]
![G{$R0{PKB V2O7(NM@ }S_X](https://user-images.githubusercontent.com/103358064/231946893-e8963cbd-0c04-4b87-97eb-675a5b49e279.png)
为什么 1.19.3 被归类到了不常用版本

**Comment by WForst-Breeze** [comment, marked as answer]
可能是因为您已经安装了 **同一大版本下** 版本号高于 1.19.3 的游戏核心，此时会折叠较低版本的游戏并归为不常用版本。如：安装了 1.19.4 后，会折叠 1.19.3，而不会折叠 1.12.2。并不是说您不经常启动就给您归到了不常用版本中去了
![image](https://user-images.githubusercontent.com/110760354/232014533-dcdac7fe-e009-40c6-bba6-a1c84c144369.png)

**Reply by Chiloven945** [reply]
这个我还不知道，抱歉（

**Comment by iPodtouch6557** [comment]
~其实这个在启动时的 你知道吗 里面有的，多启动几次就有了~
~像我在拿到内测版时已经启动了不下150次了，拿到铁杆粉的同时就知道这玩意了~
